spec.html

```diff
@@ -24084,6 +24084,7 @@ <h1>Runtime Semantics: FinishDynamicImport ( _referencingScriptOrModule_, _speci
         <p>The abstract operation FinishDynamicImport takes arguments _referencingScriptOrModule_, _specifier_, _promiseCapability_, and _completion_. FinishDynamicImport completes the process of a dynamic import originally started by an <emu-xref href="#sec-import-calls">`import()`</emu-xref> call, resolving or rejecting the promise returned by that call as appropriate according to _completion_. It is performed by host environments as part of HostImportModuleDynamically. It performs the following steps when called:</p>
 
         <emu-alg>
+          1. Assert: _promiseCapability_ is a PromiseCapability Record.
```

Comment:
A few of these are making assertions on inputs to AOs as the first step. We usually just put these in parentheticals following the alias name in the summary.

Reply:
i don't think there's a clear winner right now; both explicit and implicit assertions are possible consistent outcomes. i can make the change tho if you think it's important?

Reply:
I prefer the implicit ones, but since both styles exist currently I'm OK with having explicit ones for now.

(I think it's easier to read simple types out of the argument lists, and I don't think explicit assertions add any value once they're in the argument lists.)

Reply:
For things that are simple types, I tend to agree; for things that are, for example, non-negative integers, or something more subtle, it seems like explicit ones might make more sense.

Reply:
Yeah. Here it's a named record type, which seems simple enough to me - e.g. we already use [Realm Record](https://tc39.es/ecma262/#sec-addrestrictedfunctionproperties). and [Cyclic Module Record](https://tc39.es/ecma262/#sec-InnerModuleLinking) as a types in an argument list.

Reply:
given michael's +1 for implicit and your +0 for it, i'll go ahead and change it in this PR.